File: invesdwin-context-parent/invesdwin-context-javascript/src/test/java/de/invesdwin/context/javascript/tests/InputsAndResultsTestLong.js
print("getLong")
if(typeof getLong !== 'undefined')
	throw "getLong already defined!"
getLong = putLong
getLongType = typeof getLong
print(getLongType)
print(getLong)
if(getLongType !== 'number')
	throw "getLong not number!"

print("getLongVector")
if(typeof getLongVector !== 'undefined')
	throw "getLongVector already defined!"
getLongVector = putLongVector
getLongVectorType = typeof getLongVector[0]
print(getLongVectorType)
print(getLongVector)
if(getLongVectorType !== 'number')
	throw "getLongVector not number!"

print("getLongVectorAsList")
if(typeof getLongVectorAsList !== 'undefined')
	throw "getLongVectorAsList already defined!"
getLongVectorAsList = putLongVectorAsList
getLongVectorAsListType = typeof getLongVectorAsList[0]
print(getLongVectorAsListType)
print(getLongVectorAsList)
if(getLongVectorAsListType !== 'number')
	throw "getLongVectorAsList not number!"

print("getLongMatrix")
if(typeof getLongMatrix !== 'undefined')
	throw "getLongMatrix already defined!"
getLongMatrix = putLongMatrix
getLongMatrixType = typeof getLongMatrix[0][0]
print(getLongMatrixType)
print(getLongMatrix)
if(getLongMatrixType !== 'number')
	throw "getLongMatrix not number!"

print("getLongMatrixAsList")
if(typeof getLongMatrixAsList !== 'undefined')
	throw "getLongMatrixAsList already defined!"
getLongMatrixAsList = putLongMatrixAsList
getLongMatrixAsListType = typeof getLongMatrixAsList[0][0]
print(getLongMatrixAsListType)
print(getLongMatrixAsList)
if(getLongMatrixAsListType !== 'number')
	throw "getLongMatrixAsList not number!"

File: invesdwin-context-parent/invesdwin-context-jruby/src/test/java/de/invesdwin/context/jruby/tests/InputsAndResultsTestLong.rb
puts("getLong")
if (defined?(getLong)).nil?
	raise "getLong already defined!"
end
getLong = putLong
getLongType = getLong.class
puts(getLongType)
puts(getLong)
if(getLongType != 'number')
	raise "getLong not number!"
end

puts("getLongVector")
if (defined?(getLongVector)).nil?
	raise "getLongVector already defined!"
end
getLongVector = putLongVector
getLongVectorType = getLongVector[0].class
puts(getLongVectorType)
puts(getLongVector)
if(getLongVectorType != 'number')
	raise "getLongVector not number!"
end

puts("getLongVectorAsList")
if (defined?(getLongVectorAsList)).nil?
	raise "getLongVectorAsList already defined!"
end
getLongVectorAsList = putLongVectorAsList
getLongVectorAsListType = getLongVectorAsList[0].class
puts(getLongVectorAsListType)
puts(getLongVectorAsList)
if(getLongVectorAsListType != 'number')
	raise "getLongVectorAsList not number!"
end

puts("getLongMatrix")
if (defined?(getLongMatrix)).nil?
	raise "getLongMatrix already defined!"
end
getLongMatrix = putLongMatrix
getLongMatrixType = getLongMatrix[0][0].class
puts(getLongMatrixType)
puts(getLongMatrix)
if(getLongMatrixType != 'number')
	raise "getLongMatrix not number!"
end

puts("getLongMatrixAsList")
if (defined?(getLongMatrixAsList)).nil?
	raise "getLongMatrixAsList already defined!"
end
getLongMatrixAsList = putLongMatrixAsList
getLongMatrixAsListType = getLongMatrixAsList[0][0].class
puts(getLongMatrixAsListType)
puts(getLongMatrixAsList)
if(getLongMatrixAsListType != 'number')
	raise "getLongMatrixAsList not number!"
end
